apparmor.d/groups/browsers/firefox

```diff
@@ -9,7 +9,8 @@ abi <abi/3.0>,
 
 include <tunables/global>
 
-@{MOZ_LIBDIR} = /{usr/,}lib/firefox{,-esr}
+@{MOZ_LIBDIR}  = /{usr/,}lib/firefox{,-esr}
+@{MOZ_LIBDIR} += /opt/firefox{,-esr}
```

Comment:
Out of curiosity, how did you install Firefox to get it in /opt ?

Reply:
Manual installation from official tar. It's a natural place for third-party applications and present as example on many FF guides.